.github/workflows/ci.yml

```diff
@@ -801,6 +801,30 @@ jobs:
         run: breeze ci fix-ownership
         if: always()
 
+  tests-aws-async-provider:
+    timeout-minutes: 50
+    name: "Pytest for AWS Async Provider"
+    runs-on: "${{needs.build-info.outputs.runs-on}}"
+    needs: [build-info, wait-for-ci-images]
+    if: needs.build-info.outputs.run-tests == 'true'
```

Comment:
I guess it will run even if no changes in AWS Provider

Reply:
yes, what we should do here?

Reply:
I think we should we run only on Amazon changes, I guess @potiuk might suggest as original autor of selective check for providers